### DISCUSSION TITLE
FE: Cannot find module ../generated/*analytics, entity, etc..*

### DISCUSSION THREAD

**Discussion by nightrunner9** [open-metadata/OpenMetadata#18184] [Q&A]
Hello everyone! I'm trying to run the UI locally but I'm getting the following error:  "Cannot find module ../generated/*and any packages*". 
I'm following the instructions from the guide : https://docs.open-metadata.org/latest/developers/contribute/build-code-and-run-tests/openmetadata-ui. 

Node: 20.14.0
yarn: 1.22.22

For example:
<img width="903" alt="Снимок экрана 2024-10-09 в 12 12 49" src="https://github.com/user-attachments/assets/2ffe8214-5a7d-4964-a303-10c37484994a">

Please help. Thank you !

**Comment by Prajwal214** [comment]
@nightrunner9 Try using [Node 18.x](https://nodejs.org/en/download/) Version
https://docs.open-metadata.org/latest/developers/contribute/build-code-and-run-tests/prerequisites#prerequisites

**Reply by SpuRY** [reply]
same  problem with node 18.x

**Comment by SpuRY** [comment]
follow this directions 
https://docs.open-metadata.org/latest/developers/contribute/build-code-and-run-tests/generate-typescript-types-from-json-schema
